homeassistant/components/notify/html5.py

```diff
@@ -131,29 +140,79 @@ def post(self, request):
         return self.json_message("Push notification subscriber registered.")
 
 
+class HTML5PushCallbackView(HomeAssistantView):
```

Comment:
I think it's nice to have a generic callback mechanism for the notify component based on tags. Leaving it up to the service implementations how to forward the data to the device etc.

But that way you could call a group with an action `open_door`, have it go to HTML5 + iOS and you would be able to listen to 1 type of event for callback if you need to act.

Could be as simple that we pass in something like this:

```yaml
actions:
  open_door:
    icon: bla
    title: bla
```

With `open_door` being the tag and the value being a dict with some standardized options and also include platform specific options?